src/app/components/elements/cards/QuestionCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+import { useExpandContent } from "../markup/portals/Tables";
+import { useStatefulElementRef } from "../markup/portals/utils";
+import { Button, Card, CardBody, Row } from "reactstrap";
+import classnames from "classnames";
+import { IsaacContent } from "../../content/IsaacContent";
+import { IsaacQuestionPageDTO } from "../../../../IsaacApiTypes";
+
+const GoToQuestionFinder = () => (
+  <>
+    <h2>All done! Want more questions?</h2>
+    <Row className="w-75 mx-auto mt-5 mb-3">
+      <Button href="/gameboards/new" className="btn-block btn-lg btn-primary">
+        Go to Question Finder
+      </Button>
+    </Row>
+  </>
+);
+interface QuestionCardProps {
+  setExpanded: (expanded: boolean) => void;
+  questionData: IsaacQuestionPageDTO[];
+}
+
+const QuestionCard = ({ setExpanded, questionData }: QuestionCardProps) => {
+  const [expandRef, updateExpandRef] = useStatefulElementRef<HTMLDivElement>();
+  const { expandButton, outerClasses, expanded } = useExpandContent(true, expandRef);
+  const [question, setQuestion] = useState<number>(0);
+  const isMoreQuestions = question < 5;
```

Comment:
This could just be me, but `isMoreQuestions` sounds a bit strange. What about something like `moreQuestionsAvailable`? Or `moreQuestionsPending`?